src/game.js

```diff
@@ -0,0 +1,112 @@
+import Tile from './tiles';
+
+class Game {
+  constructor(size) {
+    const tilesWin = [];
+    for (let i = 0; i < size * size - 1; i++) {
```

Comment:
I would suggest to move all the logic in separate methods, our constuctor should look clean and just set default properties when new instance of class is initialized